curriculum/schema/challengeSchema.js

```diff
@@ -44,7 +44,12 @@ const schema = Joi.object()
       indexjsx: fileJoi
     }),
     guideUrl: Joi.string().uri({ scheme: 'https' }),
-    helpCategory: Joi.valid('JavaScript', 'HTML-CSS', 'Python'),
+    helpCategory: Joi.valid(
+      'JavaScript',
+      'HTML-CSS',
+      'Python',
+      'Relational Databases'
```

Comment:
Silly question: Is the helpCategory allowed to have whitespace?

Reply:
Not sure - I wondered that myself.